src/goto-instrument/cover.h

```diff
@@ -22,9 +22,17 @@ class message_handlert;
 class cmdlinet;
 class optionst;
 
-#define OPT_COVER "(cover):"
-#define HELP_COVER \
-  " --cover CC                   create test-suite with coverage criterion CC\n"
+#define OPT_COVER                                                              \
+  "(cover):"                                                                   \
+  "(cover-failed-assertions)"
+
+#define HELP_COVER                                                             \
+  " --cover CC                   create test-suite with coverage criterion "   \
+  "CC\n"                                                                       \
+  " --cover-failed-assertions    do not stop coverage checking at failed "     \
```

Comment:
:pick: Because of the preceding refactor, this option now applies to `jdiff` and `goto-diff` as well as `cbmc`. This means that we should probably test that it works with those entry points.